.github/workflows/ci.yml

```diff
@@ -62,6 +62,7 @@ jobs:
     with:
       target: x86_64-unknown-linux-gnu
       profile: "debug"
+      native: ${{ needs.get-runner-labels.outputs.LINUX_RUNNER_LABELS == '"ubuntu-latest"' }}
```

Comment:
We can use `${{ startsWith(runner.name, 'GitHub Actions') }}` to check whether it runs in github runner.

Reply:
@jerrykingxyz can you check again?

Reply:
@jerrykingxyz cc